site/src/components/popover.md

```diff
@@ -34,74 +34,130 @@ The popover component allows content to be placed in the popup that appears afte
-	<summary class="button">Popover aligned to the left</summary>
-	<div class="popover-content">Popover content.</div>
-</details>
-
-<details class="popover popover-center margin-xxs">
-	<summary id="test-popover-summary" class="button">Popover centered</summary>
-	<div id="test-popover-content" class="popover-content">Popover content.</div>
-</details>
-
-<details class="popover popover-right margin-xxs">
-	<summary class="button">Popover aligned to the right</summary>
-	<div class="popover-content">Popover content.</div>
+	<summary class="button">Popover</summary>
+	<div class="popover-content popover-caret">
+		Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+		labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco
+		laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
+		voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat
+		non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+	</div>
 </details>
 ```
 
-### Vertical alignment
+## Fixed positioning
 
-The popover can also show up above the button.
+You can also fix the position of the popover above the button, which will disable the vertical dynamic positioning of the popover content. The popover will still dynamically position itself horizontally to avoid overflow.
 
 ```html
-<details class="popover popover-top margin-xxs">
-	<summary class="button">Popover on top</summary>
-	<div class="popover-content">Popover content.</div>
-</details>
+<div class="display-flex justify-content-center">
+	<details class="popover margin-xxs">
+		<summary class="button">Popover top</summary>
+		<div class="popover-content popover-top">
+			Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+			labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco
+			laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
+			voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat
+			non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+		</div>
+	</details>
+</div>
 ```
 
-Vertical and horizontal alignment can be used together.
+By default, the popover content is centered with the popover button. You can use the `popover-left` or `popover-right` classes on the `.popover-content` element to align the popover content with the left or right edge of the button.
 
 ```html
-<details class="popover popover-top margin-xxs">
-	<summary class="button">Popover aligned to the left</summary>
-	<div class="popover-content">Popover content.</div>
-</details>
+<div class="display-flex justify-content-space-around">
+	<details class="popover margin-xxs">
+		<summary class="button">Left-aligned</summary>
+		<div class="popover-content popover-left popover-caret">
```

Comment:
Do we really need a caret? It looks weird and has a different thickness 

![image](https://github.com/user-attachments/assets/0da86952-7582-4abb-903b-a494122b1b9c)

![image](https://github.com/user-attachments/assets/0f6bff06-a4f0-41b4-a851-656de1e36870)

Reply:
I used the chevron, does it look better now? alignment is still difficult to have it look perfectly seamless